src/rprblender/properties/render.py

```diff
@@ -493,6 +499,11 @@ def init_rpr_context(self, rpr_context, is_final_engine=True, use_gl_interop=Fal
             if float(mac_vers_major) >= 14:
                 rpr_context.set_parameter(pyrpr.CONTEXT_METAL_PERFORMANCE_SHADER, 1)
 
+        # enable texture cache
+        if not os.path.isdir(self.texture_cache_dir):
```

Comment:
There could be file with such name, the os.path.isdir will return False. The mkdir will fail in this case.

Reply:
eh.... I mean that's a pretty weird situation.  The file dir selector doesn't let you choose a single file.